roc/receiver.go

```diff
@@ -237,7 +242,12 @@ func OpenReceiver(context *Context, config ReceiverConfig) (*Receiver, error) {
 // calling Receiver.Connect() for the interface.
 //
 // Automatically initializes slot with given index if it's used first time.
-func (r *Receiver) SetMulticastGroup(slot Slot, iface Interface, ip string) error {
+func (r *Receiver) SetMulticastGroup(slot Slot, iface Interface, ip string) (err error) {
+	logWrite(LogDebug,
+		"entering Receiver.SetMulticastGroup(): slot=%v iface=%v ip=%v", slot, iface, ip,
+	)
```

Comment:
nit: missing receiver/sender %p in "entering" message here and in many other places.